integrate-kobiton-travisci.md

```diff
@@ -0,0 +1,144 @@
+
+1. Clone this repository https://github.com/kobiton/Travis-HockeyApp-Appium-ReactNative/tree/master
+2. `cd` into `sample` folder.
+3. Create a new GitHub repository and synchronize it with Travis CI.
+4. Push content of `sample` folder to the newly created repository.
+5. Follow the next sections to get `desiredCaps` variable and replace to your script for running automation test.
+
+# B. Setup
+
+  **Purposes**
+
+  1. Securing and using Kobiton username and API key for every build in Travis CI.
+  2. Comsuming Kobiton device's Desired Capabilities to automation test script. 
+   
+## 1. Setting environment variables
+
+  1. Go to your Travis CI build project settings.
+
+  2. Add these two environment variable:
+   
+  - KOBITON_USERNAME : Your Kobiton's username.
+  - KOBITON_API_KEY : Your Kobiton's API Key.
+  
+  Please make sure to hide `KOBITON_API_KEY` value so it won't be exposed in build log.
+  
+  The result should look like below:
+  
+  ![](assets/travis_env.jpg)
+
+
+## 2. Getting desired capabilities
+
+  The desired capabilities need to be added to the automation test script in order tests to be executed on the Kobiton device.
+
+  Go to https://portal.kobiton.com and login to your Kobiton account.
+  * In the top navigation bar, select **"Devices"**.
+  
+    ![](assets/1_device_bar.jpg)
+
+  * Hover over any device you want to test with and click on the Automation settings button (the gear symbol)   
+  
+    ![](assets/2_get_device.jpg)
+
+  * You can choose your preferred language, app type. In this guideline, we will choose **"NodeJS"** for *Language* and **"Hybrid/Native from Apps"** for *App type*. Copy the code on the right to prepare for the next step.
+  
+    ![](assets/2_kobiton_device.jpg)
+
+## 3. Configuring automation test project
+  
+  Open the automation test script.
+
+  Replace the `desiredCaps` in your script with the one you got from the previous step.
+
+  **Example in Node.js:**
+
+  ```javascript
+  const desiredCaps = {
+    sessionName:        'Automation test session',
+    sessionDescription: '', 
+    deviceOrientation:  'portrait',  
+    captureScreenshots: true, 
+    app:                '<APP_URL>', 
+    deviceGroup:        'KOBITON', 
+    deviceName:         '<KOBITON_DEVICE_NAME>',
+    platformVersion:    '<KOBITON_DEVICE_VERSION>',
+    platformName:       '<DEVICE_PLATFORM_NAME>' 
+  } 
+  ```
+
+>For more information on how to run automation test on Kobiton, visit:
+  >- Kobiton documentation: https://docs.kobiton.com/automation-testing/automation-testing-with-kobiton/
+  >- Kobiton automation test script samples: https://github.com/kobiton/samples
+
+# C. Execution
+
+## 1. Run automation script on Kobiton devices
+
+Push your changes to GitHub.
+
+Your environment variables are exported from `.travis.yml`. The encrypted API key will be showed as `[secured]`.
+
+![](assets/2_travis_env.jpg)  
+
+Travis CI will install the neccessary dependencies and then run the test on Kobiton.
```

Comment:
Could you please show the very basic example of `.travis.yml` and describe a little bit. I already know that you included the `.travis.yml` in the `samples` directory but we should give the customers a basic knowledge of it.

Why: If they get test scripts from Kobiton sample, they will miss that file (`.travis.yml`).